src/com/googlecode/jmxtrans/model/naming/StringUtils.java

```diff
@@ -40,4 +40,23 @@ public static String cleanupStr(String name, boolean allowDottedKeys) {
 		clean = SPACE_PAT.matcher(clean).replaceAll("");
 		return clean;
 	}
+
+	/**
+	 * Checks if a String is whitespace, empty (""), equals empty String ("\"\"") or null.
+	 *
+	 * <pre>
+	 * StringUtils.isBlank(null)      = true
+	 * StringUtils.isBlank("")        = true
+	 * StringUtils.isBlank(" ")       = true
+	 * StringUtils.isBlank("\"\"")    = true
+	 * StringUtils.isBlank("bob")     = false
+	 * StringUtils.isBlank("  bob  ") = false
+	 * </pre>
+	 *
+	 * @param name  the String to check, may be null
+	 * @return <code>true</code> if the String is null, empty, equals the empty string or whitespace
+	 */
+	public static boolean isBlankOrEmptyString(String name) {
+		return org.apache.commons.lang.StringUtils.isBlank(name) || "\"\"".equals(name);
```

Comment:
Strange, we use Yammer metrics and I have not seen this issue. In any case, we should escape those `\"`, not make them disappear. If a lib want to add quotes in objects name, why not, we should respect this choice. We should just make sure we do not send garbage to the backends. Which means that we probably want to address this in `StringUtils.cleanupStr()`.